### PR TITLE
Fix a bug in login with cookie system

### PR DESCRIPTION
**Pull request by 01ttouch**
https://github.com/r0oth3x49/lynda-dl/issues/66#issuecomment-497932097
https://github.com/r0oth3x49/lynda-dl/issues/67

Seem like you copy the code from udemy-dl. If you log out, the cookie get destroyed and you can't login again the second time you run the code.

Did ...
@r0oth3x49 : close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close
But...
@r0oth3x49 : close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close close 🙄🙄🙄

### CODE REVIEW

**Comment by 01ttouch** [comment]
5 hours later still works fine

With the logout exists, can't even login again after 5 seconds, have to make cookie file again, what ?

**Comment by r0oth3x49** [comment]
@01ttouch the solution could be very small instead of removing codes from source. just don't waste my time in reviewing your fixes.

File: lynda/_internal.py [hunk 1]
```diff
@@ -62,9 +62,6 @@ def _fetch_course(self):
             self._short_description =       self._info['short_description']
             self._assets_count      =       self._info['assets_count']
             self._chapters          =       [InternLyndaChapter(z) for z in self._info['chapters']]
-            sys.stdout.write(fc + sd + "[" + fm + sb + "*" + fc + sd + "] : " + fg + sb + "Trying to logout now...\n")
-            self._logout()
```
Comment (r0oth3x49) [inline lynda/_internal.py:66]: a simple fix for this could be instead of removing the code
```python
if not self._cookies:
    self._logout()
```

**Review by r0oth3x49 — COMMENTED** [review]
a simple fix for this could be instead of removing the code
```python
if not self._cookies:
    self._logout()
```

**Pull request closed without merging**